src/Entity/Adherent.php

```diff
@@ -229,6 +220,23 @@ class Adherent implements UserInterface, GeoPointInterface, EncoderAwareInterfac
      */
     private $roles = [];
 
+    /**
+     * @ORM\Column(type="boolean")
```

Comment:
The column is not nullable b y default though the default should also be set at db level through the mapping:
```php
/**
 * @ORM\Column(type="boolean", options={"default"=false})
 */
```
and the migration will need to be updated.